shynet/dashboard/templates/dashboard/pages/service.html

```diff
@@ -109,8 +109,24 @@
             <tbody>
                 {% for location in stats.locations %}
                 <tr>
-                    <td class="truncate w-full max-w-0">{{location.location|default:"Unknown"|urldisplay}}</td>
-                    <td class="rf">{{location.count|intcomma}}</td>
+                    <td class="truncate w-full max-w-0 relative">
+                        <div
+                            class="absolute h-6"
+                            style="width: {{location.count|percent:stats.hit_count}}; top: 6px; left: 6px; height: calc(100% - 12px); background-color: var(--color-urge-200-fallback)"
```

Comment:
Maybe move style attributes to css file to minimize duplicate code like in [this commit](https://github.com/CasperVerswijvelt/shynet/commit/19520279de8e5f89794bb23e14dd9596ae9c3868)